.circleci/slack.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/sh -e
+#
+# Stupid and Simple Slack Notifier.
+#
+
+LC_ALL=en_US.UTF-8; export LC_ALL
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+WORKDIR=$(mktemp -d)
+trap 'rm -rf $WORKDIR' 0 1 2 3 6 14 15
+
+## Utils
+##
+
+printe() {
+    printf >&2 "%s\\n" "$@"
+}
+
+err() {
+    printf >&2 "%s: %s\\n" "$(basename "$0")" "$*"
+}
+
+_escape() {
+    printf "%s" "$1" | sed 's/\\/\\\\/g' | sed 's/"/\\"/g'
```

Comment:
Quick workaround for `CIRCLE_JOB` has some sort of newline appended to it: add `| tr -d "\n"` after final sed.